src/design/plone/theme/theme/sass/components/_tiles_base.scss

```diff
@@ -152,47 +153,3 @@
         }
```

Comment:
La parte qui sopra con tutte le larghezze fissate forse va un po' sistemata. Credo vada in conflitto con quanto definito ad inizio file, dove è gestita con i mixin di bootstrap che sono più completi. Capisco anche che se non si mette una larghezza qui, allora avranno sempre 100% per via del ``width: 100%`` sul ``tileWrapper``, quindi, se il mio ragionamento ha senso, direi che bisogna farci una pensata bene per strutturarlo a modo.

Reply:
Hai ragione.
Ora l'ho aggiornato mantenendo la logica attuale ma eliminando il doppione.
Facciamoci eventualmente una pensata

Reply:
Hai ragione.
Ora l'ho aggiornato mantenendo la logica attuale ma eliminando il doppione.
Facciamoci eventualmente una pensata